ability_item_usage_life_stealer.lua

```diff
@@ -457,23 +456,23 @@ local consumeThink = function()
 	return 0
 end
 
-Consider[5] = function()
-	if AbilitiesReal[5]:GetName() == "life_stealer_consume" then
-		return consumeThink()
-	end
-	if AbilitiesReal[5]:GetName() == "life_stealer_infest" then
-		return infestThink()
+-- Some previous code seems to hint these can switch places.
+-- This might actually be unnecessary
```

Comment:
The ability position change here is very interesting, it seems to be a bug in Dota2 itself.

Reply:
Not sure if you'd like me to change something here (maybe the comment?).  If these two abilities do swap places for whatever reason, this updated code should handle it better than the previous code.

Reply:
> Not sure if you'd like me to change something here (maybe the comment?). If these two abilities do swap places for whatever reason, this updated code should handle it better than the previous code.

No changes needed, I'm just curious about this kind of bug.